environments/daml-sdk-env/build/3_configdaml.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+echo "PATH=\$PATH:~/.daml/bin" >> ~/.bashrc
+source ~/.bashrc
+
+# uninstall unnecessary VSCode extensions
+rm -rf /opt/.katacodacode/extensions/redhat.vscode-openshift-connector-0.0.19/;
+rm -rf /tmp/.katacodacode/extensions/redhat.vscode-openshift-connector-0.0.19/;
+
+# install daml VSCode extension
+cd /tmp && wget http://assets.joinscrapbook.com/daml/DigitalAssetHoldingsLLC.daml-0.13.55.vsix;
```

Comment:
There should be no need to download it, the extension is bundled in the SDK and can be found in `~/.daml/sdk/0.13.55/studio/daml-bundled.vsix`
```suggestion
cd /tmp && cp ~/.daml/sdk/0.13.55/studio/daml-bundled.vsix .;
```